static/elements/chromedash-feature.js

```diff
@@ -0,0 +1,492 @@
+    if (!this.feature) return undefined;
+    const vendors = (this.feature.browsers.ff.view.val +
+                   this.feature.browsers.edge.view.val +
+                   this.feature.browsers.safari.view.val) / 3;
+    const webdevs = this.feature.browsers.webdev.view.val;
+    const standards = this.feature.standards.status.val;
+    return vendors + webdevs + standards;
+  }
+
+  set _interopRisk(value) {/* Never called */}
+
+  get _isDeprecated() {
+    const DEPRECATED_STATUSES = ['Deprecated', 'No longer pursuing'];
+    return DEPRECATED_STATUSES.includes(
+      this.feature.browsers.chrome.status.text);
+  }
+
+  set _isDeprecated(value) {/* Never called */}
+
+  get _hasDocLinks() {
+    return this.feature.resources.docs &&
+        this.feature.resources.docs.length > 0;
+  }
+
+  set _hasDocLinks(value) {/* Never called */}
+
+  get _hasSampleLinks() {
+    return this.feature.resources.samples &&
+        this.feature.resources.samples.length > 0;
+  }
+
+  set _hasSampleLinks(value) {/* Never called */}
+
+  get _commentHtml() {
+    return urlize(this.feature.comments,
+      {target: '_blank', trim: 'www', autoescape: true});
+  }
+
+  set _commentHtml(value) {/* Never called */}
+
+  _fireEvent(eventName, detail) {
+    let event = new CustomEvent(eventName, {detail});
+    this.dispatchEvent(event);
+  }
+
+  toggle(e) {
+    // Don't toggle panel if tooltip or link is being clicked.
+    const target = e.currentTarget;
+    if (target.classList.contains('tooltip') || 'tooltip' in target.dataset ||
+        target.tagName == 'A' || target.tagName == 'CHROMEDASH-MULTI-LINKS') {
+      return;
+    }
+
+    this.open = !this.open;
+
+    // Handled in `chromedash-featurelist`
+    this._fireEvent('feature-toggled', {
+      feature: this.feature,
+      open: this.open,
+    });
+  }
+
+  categoryFilter(e) {
+    e.stopPropagation();
+    e.preventDefault();
+    // Listened in `templates/features.html`
+    this._fireEvent('filter-category', {val: e.currentTarget.innerText});
+  }
+
+  filterByOwner(e) {
+    e.stopPropagation();
+    e.preventDefault();
+    // Listened in `templates/features.html`
+    this._fireEvent('filter-owner', {val: e.currentTarget.innerText});
+  }
+
+  filterByComponent(e) {
+    e.stopPropagation();
+    e.preventDefault();
+    // Listened in `templates/features.html`
+    this._fireEvent('filter-component', {val: e.currentTarget.innerText});
+  }
+
+  subscribeToFeature(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    const featureId = this.feature.id;
+    if (!featureId) {
+      return;
+    }
+
+    this._receivePush = !this._receivePush;
+
+    if (this._receivePush) {
+      PushNotifications.subscribeToFeature(featureId);
+    } else {
+      PushNotifications.unsubscribeFromFeature(featureId);
+    }
+  }
+
+  render() {
+    return html`
+      <link rel="stylesheet" href="/static/css/elements/chromedash-feature.css">
+
+      <div class="main-content-area">
```

Comment:
You probably don't need this element, since it looks like a wrapper on the entire shadow DOM contents. Most likely you can just style the host instead.

Reply:
Done